src/Pages/ForgetPassword/ForgetPassword.tsx

```diff
@@ -1,10 +1,94 @@
-import React from 'react'
-import './ForgetPassword.module.scss'
+import {
+  Box,
+  Button,
+  FormControl,
+  OutlinedInput,
+  Typography,
+} from "@mui/material";
+import axios from "axios";
+import { useForm } from "react-hook-form";
+import { Link, useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import baseUrl from "../../utils/Custom/Custom";
+import "./ForgetPassword.module.scss";
 
 const ForgetPassword = () => {
-  return <>
-    <div>ForgetPassword</div>
-  </>
-}
+  const navigate = useNavigate();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+  const onSubmit = (data) => {
+    axios
+      .post(`${baseUrl}/api/v0/portal/users/forgot-password`, data)
+      .then((res) => {
+        console.log("data", data);
+        toast.success("send successfully");
+        navigate("/resetPassword");
+      })
+      .catch((err) => {
+        console.log(err.response.data.message);
+        toast.error(err.message);
+      });
+  };
+  return (
+    <>
+      <Box component="div">
+        {" "}
+        <Typography variant="h4" component="h4" sx={{ padding: "20px" }}>
+          <Box component="span" sx={{ color: "skyblue" }}>
+            Stay
+          </Box>
+          cation.
+        </Typography>
+      </Box>
+      <Box sx={{ padding: "30px 70px" }}>
+        <Box component="div">
+          <Typography variant="h4" component="h4">
```

Comment:
This shouldn't be an h4 element. It is just a normal link. Making it an h4 element will give bad page hierarchy and structure